apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/RowFilters.cs

```diff
@@ -0,0 +1,407 @@
+﻿// Copyright 2017 Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Protobuf;
+using System;
+
+namespace Google.Cloud.Bigtable.V2
+{
+    /// <summary>
+    /// Static class with convenience methods for creating various kinds of <see cref="RowFilter"/> instances.
```

Comment:
Given the factory methods here, I'd move the ones from ColumnRange to here too.

Reply:
Just to clarify, are you saying there should be 4 additional overloads to `RowFilters.ColumnRange` that take the start/end values, and named appropriately like `ColumnRangeOpen` or `ColumnRangeExclusive` (depending on which naming we go with)? Also, when you say 'move' them, do you mean they should no longer appear as factory methods on `ColumnRange` or that they should exist in both places?

Also, I assume the same deicsions would apply to `ValueRange`, right?

Reply:
I'm saying that if I know that there are factory methods in `RowFilters` to create row filters, I'm not sure I'd think to look elsewhere as well. But this is very much from the perspective of a review without reading any docs etc. I can see that there's a lot of benefit in using the ColumnRange and ValueRange types to give extra information in the type name - I hadn't appreciated that there'd be both, and they both create row filters...

Reply:
Ok, I'm going to leave as is for now, but add TODOs for these two filter types to see if we should have methods taking start/end values for convenience.